learning_resources/views.py

```diff
@@ -113,6 +113,7 @@ def get_queryset(self) -> QuerySet:
         """
         return self._get_base_queryset()
 
+    @extend_schema(responses=LearningResourceSerializer(many=True))
     @action(methods=["GET"], detail=False, name="New Resources")
```

Comment:
From the drf-spectacular [FAQ](https://drf-spectacular.readthedocs.io/en/latest/faq.html#i-m-using-action-detail-false-but-the-response-schema-is-not-a-list):

> I'm using `@action(detail=False)` but the response schema is not a list
> -------------------------------------------------------------------------
>
> `detail=True/False` only specifies whether the action should be routed at `x/{id}/action` or `x/action`. The `detail` parameter in itself makes no statement about the action's response. Also note that the default for underspecified endpoints is a non-list response. To signal a listed response, you can use `@extend_schema(responses=XSerializer(many=True))`.

So since you could create non-list responses when `detail=False`, drf-spectacular doesn't automatically listify them.